### ISSUE TITLE
Clarify latent.vars documentation in FindMarkers and FindAllMarkers

### ISSUE THREAD

**Issue by b-dawes** [satijalab/seurat#386]
This is similar to another documentation clarification I posted about a bit ago. I noticed that the latent.vars argument is only applied with the negbinom, poisson, and MAST tests. Could a note about this be added to the documentation? Also it could be useful to give a warning if latent.vars is provided but is not supported by the chosen test.

**Comment by satijalab** [comment]
Yes, I agree. We'll fix this and the previous issue in the next release (either will issue a warning, make clear in the docs, or both).
